packages/quri_parts/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[build-system]
+requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning", "setuptools"]
+build-backend = "poetry_dynamic_versioning.backend"
+
+
+
+[tool.poetry]
+name = "jijmodeling-transpiler-quantum-quri-parts"
+version = "0.1.0rc2"
+description = "JijModeling Transpiler for Quri-parts"
```

Comment:
The correct name is QURI Parts.

Reply:
https://github.com/Jij-Inc/JijModelingTranspilerQuantum/blob/7a4630de56677aab4bab4ad00f80ca9824bfefc0/packages/quri_parts/pyproject.toml#L10
Changed!

Reply:
P is large.
https://quri-parts.qunasys.com/index.html#quri-parts

Reply:
Fixed